webapp/src/app/models/multi-entity-animation.ts

```diff
@@ -0,0 +1,57 @@
+import { Point3 } from './cross-code-map';
+import { TileSheet } from './tile-sheet';
+
+export interface MultiEntityAnimation {
+    DOCTYPE: 'MULTI_ENTITY_ANIMATION';
+
+    anims: Record<string, EntityAnim>;
+    baseSize: Point3;
+    namedSheets?: Record<string, TileSheet>;
```

Comment:
isn't this type just `{ [key: string]: TileSheet }`?

Reply:
That is correct - `type Record<A, B> = { [key: A]: B }`. Should I change it?

Reply:
I don't really care what we use. I just never saw `Record`used before